### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by viabadia**

### CODE REVIEW

**Comment by aleixcam** [comment]
where's the code?

**Comment by viabadia** [comment]
I dont have code. in five days will be uploaded, sorry.

File: staff/victor-abadia/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                                 
+                                </tr>
+                                </tbody>
+                        </table>
+        </nav>
+                        <hr>
+        <nav id="footer"><h2>Contact</h2>
+                        <ul>hello@nomennescio.com | +34 937777777</ul>   
+        </nav>                                                      
+                              </p>
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:106]: what's that ```<p>``` here?

File: staff/victor-abadia/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                                    <td>Lorem Ipsum is simply dummy text</td>
+                                    <td>2008</td>
+                                 
+                                </tr>
+                                </tbody>
+                        </table>
+        </nav>
+                        <hr>
+        <nav id="footer"><h2>Contact</h2>
+                        <ul>hello@nomennescio.com | +34 937777777</ul>
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:104]: ```<ul>``` without ```<li>```s? also the email and the phone are different informations

File: staff/victor-abadia/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                                    <td>Lorem Ipsum</td>
+                                    <td>Lorem Ipsum is simply dummy text</td>
+                                    <td>2008</td>
+                                 
+                                </tr>
+                                </tbody>
+                        </table>
+        </nav>
+                        <hr>
+        <nav id="footer"><h2>Contact</h2>
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:103]: ```<nav>``` is used for menus, what you are trying to do here is a ```<footer>```

File: staff/victor-abadia/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        <nav id="know-hows"><h2>Know-hows</h2>
+                        <ul>
+                                <li><strong>Web developer:</strong> HTML5, CSS3, JavaScript, PHP, SEO, Node.js, MySQL, AngularJS, jQuery</li>
+                                <li><strong>Web Analytics:</strong> Google Analytics, Mixpanel, segment.io</li>
+                                <li><strong>Management</strong>: SCRUM, community management and event organization, team management</li>
+                                <li><strong>Entrepreneurship:</strong> startups, Lean Startup, etc.</li>
+                        </ul>
+        </nav>
+                        <hr>
+        <nav id="education"><h2>Education</h2>
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:70]: ```<nav>``` is used for menus, what you are trying to do here is a ```<section>```

File: staff/victor-abadia/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Nomen Nescio - Curriculum Vitae</title>
+</head>
+<body>
+        <nav id="header">
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:10]: ```<nav>``` is used for menus, what you are trying to do here is a ```<header>```

File: staff/victor-abadia/cv/index.html [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+                                 
+                                </tr>
+                                <tr>
+                                    <td>Lorem Ipsum</td>
+                                    <td>Lorem Ipsum is simply dummy text</td>
+                                    <td>2018</td>
+                                 
+                                </tr>
+                                </tbody>
+                        </table>
```
Comment (aleixcam) [inline staff/victor-abadia/cv/index.html:57]: table desing is the worst thing you can do, you can use ```<article>```s here

**Pull request merged**